test/nlp/test_autohf_tokenclassification.py

```diff
@@ -42,6 +46,74 @@ def test_tokenclassification():
     except requests.exceptions.HTTPError:
         return
 
+    # perf test
+    import json
+
+    with open("seqclass.log", "r") as fin:
+        for line in fin:
+            each_log = json.loads(line.strip("\n"))
+            if "validation_loss" in each_log:
+                val_loss = each_log["validation_loss"]
+                min_inter_result = min(
+                    [
+                        each_dict.get("eval_automl_metric", -sys.maxsize)
```

Comment:
When you have one -sys.maxsize in the list, the min is going to be -sys.maxsize. Is that intended?